content/en/docs/usage/certificate.md

```diff
@@ -33,14 +34,30 @@ metadata:
   name: example-com
   namespace: default
 spec:
+  # Secret names are always required.
   secretName: example-com-tls
   duration: 2160h # 90d
   renewBefore: 360h # 15d
+  organization: jetstack
```

Comment:
This should be `organizations:` and is a `[]string`

Reply:
Changed, but it is actually `organization` in the API :man_shrugging: 


```
            organization:
              description: Organization is the organization to be used on the Certificate
              type: array
              items:
                type: string
```

Reply:
🤦‍♂

Reply:
This might be a candidate to fix up in a future version...